packages/labextension/package.json

```
@@ -31,13 +31,13 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@jupyterlab/application": "^1.0",
-    "@jupyterlab/apputils": "^1.0",
-    "@jupyterlab/notebook": "^1.0"
+    "@jupyterlab/application": "^2.0.0-beta.2",
+    "@jupyterlab/apputils": "^2.0.0-beta.2",
+    "@jupyterlab/notebook": "^2.0.0-beta.2"
```

I find it a little strange to take a dependency on beta versions (I know this was in the migration guide!!). Do you think we could use
```suggestion
    "@jupyterlab/application": "^2.0",
    "@jupyterlab/apputils": "^2.0",
    "@jupyterlab/notebook": "^2.0"
```
NB: Maybe I would expect the latter to work for a longer time period...

---

Agreed, my guess is the migration [guide](https://jupyterlab.readthedocs.io/en/stable/developer/extension_migration.html) is not quite finalised (it was up before `2.0` was released and might not have been updated since).